Makefile

```diff
@@ -0,0 +1,114 @@
+.SUFFIXES: .c .cpp .o
+
+LIB = libgambatte/libgambatte.a
+SDL = gambatte_sdl/gambatte_sdl
+TEST = test/testrunner
+
+PY = python
```

Comment:
Some distros use the environment variable PYTHON to handle things. I also find the name better.
```
PYTHON ?= python
```
for example, but both ways work ofc.